src/Datagrid/Pager.php

```diff
@@ -47,14 +47,6 @@ public function getResults()
         return $this->getQuery()->execute();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getQuery()
```

Comment:
This is the same as its parent [Pager](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Datagrid/Pager.php#L564).